nodejs/week1/mealsharing_app_1/src/backend/routes/cheap-meals.js

```diff
@@ -0,0 +1,32 @@
+/* Respond with the json for all the meals (including it's reviews) that are cheap (you define what a cheap meal is) */
+
+const express = require('express');
+const app = express();
+
+// data source
+const jsonMeals = require(__dirname + '/../data/meals.json');
+const jsonReviews = require(__dirname + '/../data/reviews.json');
+
+
+const cheapMealsFiltered = jsonMeals.filter((aCheapMeal) => aCheapMeal.price < 107);
+
+cheapMealsFiltered.forEach((aMeal) => {
+
+    aMeal.review = [];
```

Comment:
You could write it in a simpler way. Check this comment for a reference:

https://github.com/krithi-kondana/hyf-homework/pull/30#discussion_r382179162

The same comment applies to other files as well 🙂

Reply:
You can write it shorter with filter:
`meal.review = reviewsJson.filter(mealReview => mealReview.mealId === meal.id)[0]`
If you have questions regarding how it works, ask me on Sunday ;)
The same code applicable to all the places you used similar logic. 
---> copy-paste ---> to be implemented :) Thank you!